config/settings.yml

```diff
@@ -935,7 +935,6 @@ mhv:
     gw_base_path: <%= ENV['mhv__rx__gw_base_path'] %>
     host: <%= ENV['mhv__rx__host'] %>
     mock: <%= ENV['mhv__rx__mock'] %>
-    use_new_api: <%= ENV['mhv__rx__use_new_api'] %>
```

Comment:
Make sure you delete this (for all envs) in parameter store after this PR is _deployed_

Reply:
👍 I see all mentions of this setting removed in vets-api